docs/includes/query-builder/QueryBuilderTest.php

```diff
@@ -62,8 +63,8 @@ public function testOrWhere(): void
     {
         // begin query orWhere
         $result = DB::connection('mongodb')
-            ->table('movies')
-            ->where('year', 1955)
+            ->collection('movies')
```

Comment:
```suggestion
            ->table('movies')
```

Reply:
I'll make the rest of these collection -> table changes in https://jira.mongodb.org/browse/DOCSP-42956